include/Rotation.h

```diff
@@ -99,12 +102,14 @@ namespace ale {
       /**
        * Rotate a vector
        *
-       * @param vector The vector to rotate. Cab be a 3 element position or 6 element state.
+       * @param vector The vector to rotate. Can be a 3 element position or 6 element state.
        * @param av The angular velocity to use when rotating state vectors. Defaults to 0.
        *
        * @return The rotated vector.
        */
-      std::vector<double> operator()(const std::vector<double>& vector, const std::vector<double>& av = {0.0, 0.0, 0.0}) const;
+      ale::Vec3d operator()(const ale::Vec3d &av) const;
```

Comment:
The name of this argument should be vector. It's just rotating a vector, not rotating specifically an angular velocity vector.